bin/core/src/startup.rs

```diff
@@ -35,6 +39,52 @@ pub async fn on_startup() {
   );
 }
 
+pub async fn run_startup_actions() {
+    let actions = match mungos::find::find_collect::<Action>(
+        &db_client().actions,
+        None,
```

Comment:
Can directly query for actions with this config using `doc! { "config.run_at_startup": true }`. Can just make this change myself after merge